extruct/opengraph.py

```diff
@@ -2,6 +2,17 @@
 import lxml.html
 
 
+_PREFIX_PATTERN = re.compile(r'\s*(\w+): ([^\s]+)')
```

Comment:
I wonder if we should be less strict regarding whitespaces after `:`; in http://ogp.me/ I found nothing about a single whitespace being required. Something like `r'\s*(\w+):\s*([^\s]+)'`?

Reply:
I've tried to find if there is any specification about that, but didn't find any. Agree, will be better to put `\s*` there :+1:

Reply:
Done